app/Http/Controllers/CampaignRegistrationController.php

```diff
@@ -53,4 +53,32 @@ public function store(Request $request)
 
         return redirect('campaign.index');
     }
+
+    public function destroy(Request $request, Campaign $campaign_registration)
```

Comment:
Ah just realized this variable name goes against our code conventions it should be
```suggestion
    public function destroy(Request $request, Campaign $campaignRegistration)
```

Can you rename it everywhere in the method?